Controller/MainController.py

```diff
@@ -258,10 +257,12 @@ def set_robot_pos_on_screen(self, bot_id, team_color, pst, theta):
         """ Modifie la position et l'orientation d'un robot sur le terrain """
         self.view_field_screen.set_bot_pos(bot_id, team_color, pst[0], pst[1], theta)
 
-    def set_field_size(self, frame_field_size):
+    def set_field_size(self, frame_geometry_field):
         """ Modifie la dimension du terrain provenant des frames de vision"""
-        QtToolBox.field_ctrl.set_field_size(frame_field_size)
+        QtToolBox.field_ctrl.set_field_size(frame_geometry_field)
 
+    def hide_ball(self):
+        self.hide_mob(None)
```

Comment:
white line cop 👮

Reply:
Also, could we call `self.view_field_screen.hide_ball()` directly from here? If I'm not mistaken, that would allow us to remove a if/else in the `hide_mob` method.
I don't like having to public methods doing the exact same thing (in this case, hiding the ball)